dex/networks/erc20/txdata.go

```diff
@@ -0,0 +1,75 @@
+// This code is available on the terms of the project LICENSE.md file,
```

Comment:
Should this file just go into `dex/networks/eth`? Neither client or server have an erc20 folder, so I think we should just get rid of the `dex/networks/erc20` folder.

Reply:
There's other stuff in the `dex/networks/erc20` folder, but do I understand that you mean that the new txdata stuff and existing params stuff could be moved to dexeth? I personally feel like we should keep the packages separate, but I really hadn't though much about it. I don't see any reason why the networks folder needs to match the structure of either client or server. I guess there might be some other advantages to combining though. I wonder how this choice might affect the future implementation of other EVM assets. If you think it's important enough, you could open an issue.